mbuild/lattice.py

```diff
@@ -34,7 +34,7 @@ def load_cif(file_or_path=None, wrap_coords=False):
         frame = my_cif[0]
 
         # convert angstroms to nanometers
-        lattice_spacing = [frame.box.Lx/10, frame.box.Ly/10, frame.box.Lz/10]
+        lattice_spacing = np.linalg.norm(np.asarray(frame.box.get_box_matrix()).T, axis=1) / 10
```

Comment:
Two questions? Why the `box_matrix` (https://garnett.readthedocs.io/en/stable/trajectory.html#garnett.trajectory.Box)?

And secondly why the transpose?

Reply:
The reason I used `box_matrix` was directly related to the issue that the lengths of the box are already converted to the euclidean coordinate system, which would be a bit of a pain to back-calculate the box lengths in lattice space. However, the magnitude of the box vectors directly provides `a, b, c` (`np.linalg.norm`). And the transpose is necessary since `garnett` stores its box information in a column-major format, while everything in mbuild is row-major.

Reply:
That all makes sense. LGTM!